website/src/components/canvas/Canvas.jsx

```diff
@@ -194,6 +199,31 @@ class Canvas extends React.Component {
     this.hasInitialized = true;
   }
 
+
+  /**
+  * Key press handler
+  * @param {Object} event is the event listener for the window
+  * @returns {void}
+  */
+  handleKeyPress(event) {
+    if(this.props.targetedId){
+      if(event.code === "Delete"){
+        this.props.dispatch(ElementActions.removeElementAndPersist(
+          this.props.targetedId, this.props.params.canvasId));
+      }
+      else if(event.ctrlKey === true && event.code === "KeyC") {
+        const targetElement = this.props.elements[this.props.targetedId];
+        FBHelper.cloneElement(this.props.params.canvasId, targetElement,
+          CC.INIT_POSITION);
+      }
+      else if(event.ctrlKey === true && event.code === "KeyV") {
+        const targetElement = this.props.elements[this.props.targetedId];
+        FBHelper.cloneElement(this.props.params.canvasId, targetElement,
+          CC.INIT_POSITION);
+      }
```

Comment:
cloning is set up to occur on both ctrl c and ctrl v here. Might just want to bind it to crtl-v (removing lines 214-218), since we don't really have a way to "copy" the element without cloning it.